zkevm-circuits/src/sig_circuit/ecdsa.rs

```diff
@@ -81,39 +81,43 @@ where
     // load required constants
     let zero = scalar_chip.load_constant(ctx, FpConfig::<F, SF>::fe_to_constant(SF::zero()));
     let one = scalar_chip.load_constant(ctx, FpConfig::<F, SF>::fe_to_constant(SF::one()));
-    let neg_one = scalar_chip.load_constant(ctx, FpConfig::<F, SF>::fe_to_constant(-SF::one()));
+    let point_at_infinity = EcPoint::construct(
+        ecc_chip
+            .field_chip()
+            .load_constant(ctx, fe_to_biguint(&CF::zero())),
+        ecc_chip
+            .field_chip()
+            .load_constant(ctx, fe_to_biguint(&CF::zero())),
+    );
 
     // compute u1 = m * s^{-1} mod n
     let s2 = scalar_chip.select(ctx, &one, s, &s_is_zero);
```

Comment:
Based on the rest of the nomenclature, we should also call this `s_prime` instead. The uniformity will make it easier to understand